yarn-project/aztec-node/src/aztec-node/server.ts

```diff
@@ -164,11 +164,6 @@ export class AztecNodeService implements AztecNode, Traceable {
 
     const archiver = await createArchiver(config, blobSinkClient, { blockUntilSync: true }, telemetry);
 
-    // we identify the P2P transaction protocol by using the rollup contract address.
-    // this may well change in future
-    const rollupAddress = config.l1Contracts.rollupAddress;
-    config.transactionProtocol = `/aztec/tx/${rollupAddress.toString()}`;
```

Comment:
in the interest of cleaning up config i can remove this from config in a follow up